db/schema.rb

```diff
@@ -44,14 +44,15 @@
   end
 
   create_table "language_skills", force: :cascade do |t|
-    t.bigint "client_id"
+    t.bigint "languageable_id"
     t.string "language"
     t.string "level"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.datetime "deleted_at"
-    t.index ["client_id"], name: "index_language_skills_on_client_id"
+    t.string "languageable_type"
     t.index ["deleted_at"], name: "index_language_skills_on_deleted_at"
+    t.index ["languageable_id"], name: "index_language_skills_on_languageable_id"
```

Comment:
The index for polymorphic association is a little bit different. It should look like:
```ruby
    t.index ["languageable_type", "languageable_id"], name: "index_ language_skills_on_languageable_type_and_languageable_id"
```
So you need to make a migration which contains something like this:
```ruby
add_index(: language_skills, [:languageable_type, :languageable_id], unique: true)
```
The name should then be correctly generated